data/jobs.ts

```diff
@@ -10,6 +10,34 @@ export type Job = {
 };
 
 export const jobs: Job[] = [
+  {
+    title: "Junior Sector Specialist (Business & Public Services PM)",
+    company: "FactSet",
+    desc: "The Junior Sector Specialist will provide research and support of the FactSet Revere products. Quantitative and qualitative analysis of companies and levels within the sector, and along with collaboratively work with the broader content organization including the Revere and Private Markets team, sales, consulting, and other FactSet SBUs.",
```

Comment:
~~and along with collaboratively work~~ 
and work collaboratively

Reply:
Good catch!